app_dart/lib/src/service/scheduler.dart

```diff
@@ -128,7 +128,22 @@ class Scheduler {
 
     final CiYaml ciYaml = await getCiYaml(commit);
     final List<Target> initialTargets = ciYaml.getInitialTargets(ciYaml.postsubmitTargets);
-    final List<Task> tasks = targetsToTask(commit, initialTargets).toList();
+    List<Task> tasks = targetsToTask(commit, initialTargets).toList();
```

Comment:
error - A value of type 'List<FullTask>' can't be assigned to a variable of type 'List<Task>' at lib/src/service/scheduler.dart:146:13 - (invalid_assignment)

Reply:
See L146 for the comment. This is a draft in case anyone has a cycle to fix it and add a test